cmd/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/openshift/assisted-service/internal/dns"
 	"github.com/openshift/assisted-service/internal/domains"
 	"github.com/openshift/assisted-service/internal/events"
+	"github.com/openshift/assisted-service/internal/events/event"
```

Comment:
what's the difference between the two ?
```	
"github.com/openshift/assisted-service/internal/events"
"github.com/openshift/assisted-service/internal/events/event"
```

Reply:
I plan to write a detailed commit-msg that will also include this, will write it also here:
Adding a user check (via the identity pkg) for `V2GetEvents()` caused  a cyclic import:

```go
+ find /home/nmagnezi/go/src/github.com/assisted-service -name 'mock_*.go' -type f -delete
++ go list ./...
++ grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi'
package github.com/openshift/assisted-service/cmd
	imports github.com/openshift/assisted-service/internal/assistedserviceiso
	imports github.com/openshift/assisted-service/internal/cluster/validations
	imports github.com/openshift/assisted-service/pkg/auth
	imports github.com/openshift/assisted-service/pkg/ocm
	imports github.com/openshift/assisted-service/internal/metrics
	imports github.com/openshift/assisted-service/internal/events
	imports github.com/openshift/assisted-service/internal/identity
	imports github.com/openshift/assisted-service/pkg/ocm: import cycle not allowed
+ go generate
```

To overcome this, the implementation (that imports identity pkg) is now on a separate pkg from the Interface.